persists/backend_server/server.bal

```diff
@@ -0,0 +1,66 @@
+import ballerina/http;
+import ballerina/persist;
+import ballerina/random;
+
+final Client ordersDatabase = check new ();
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"]
+    }
+}
+service /sales on new http:Listener(9090) {
+
+    // Example: http://localhost:9090/sales/orders
+    isolated resource function post orders(Order orderEntry) returns http:Ok|http:InternalServerError|http:BadRequest|error {
```

Comment:
I think its better if we can have a consistency on how we order these resource function. IMO something like below would be nice,
```
/path1
1. get
2. post
3. put

/path2
1. get
2. post
3. put
```

Reply:
I added `post` first and then `get. Because `post` resources have heavy logics than `get`.
wdyt?

Reply:
updated.